src/test/java/pizzashop/IntegrationPaymentRepositoryInPizzaServiceTest.java

```diff
@@ -0,0 +1,120 @@
+package pizzashop;
+
+import org.junit.jupiter.api.*;
+import pizzashop.model.Payment;
+import pizzashop.model.PaymentType;
+import pizzashop.repository.MenuRepository;
+import pizzashop.repository.PaymentRepository;
+import pizzashop.service.PizzaService;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+/*
+    * Integration of PaymentRepository in PizzaService.
+ */
+public class IntegrationPaymentRepositoryInPizzaServiceTest {
+    private static PizzaService service;
+    private final static String pathToPayments = "data/payments.txt";
+    private static List<String> initialPayments;
+    private static File filePayments;
+
+
+
+    @Disabled
+    private static List<String> readPayments()
+    {
+        List<String> lines = new ArrayList<>();
+
+        try (BufferedReader br = new BufferedReader(new FileReader(filePayments))) {
+            lines.addAll(br.lines().collect(Collectors.toList()));
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        return lines;
+    }
```

Comment:
ai mai facut aceeasi chestie in PaymentRepoUnit si in celalalt integration. de ce refaci astea din repo si de 3 ori acelasi cod in 3 locuri diferite?